api_v3/services/BaseEntryService.php

```diff
@@ -851,4 +854,19 @@ function cloneAction( $entryId )
 
 		return $this->getEntry($clonedEntry->getId());
 	}
+	
```

Comment:
Please move this method to KalturaEntryService and call it also from KalturaEntryService::add instead of having duplicated code.